test/runtime/Policies.go

```diff
@@ -486,6 +487,62 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 		connectivityTest(allRequests, helpers.App2, helpers.Httpd1, true)
 	})
 
+	It("Tests Endpoint Connectivity Functions After Daemon Configuration Is Updated", func() {
+		httpd1DockerNetworking, err := vm.ContainerInspectNet(helpers.Httpd1)
+		Expect(err).ToNot(HaveOccurred(), "unable to get container networking metadata for %s", helpers.Httpd1)
+
+		By("Importing policy and waiting for revision to increase for endpoints")
+		_, err = vm.PolicyImportAndWait(vm.GetFullPath(policiesL7JSON), helpers.HelperTimeout)
```

Comment:
Why a new policy is needed? It was not before no?

Reply:
What do you mean by a 'new policy'? This is a policy that is already in the repository. Also, what was 'not needed' before? I don't understand. Please elaborate.

Reply:
Ok! I was not sure why the policy was in there at all. If you can add a small comment why the policy is imported will be great :-)

Reply:
Done.